llm-guard/app/scanner.py

```
@@ -0,0 +1,37 @@
+        model=config.MODEL,
+        device=-1,
+        truncation=True,
+        max_length=512,
+        top_k=None,
+    )
+    logger.info("model ready")
+
+
+def scan(text: str) -> tuple[bool, float]:
+    """Return (is_safe, injection_score) for the given text."""
+    results = _PIPE[0](text)
```

Pylint (4.0.5)

[error] _PIPE[0] is not callable

(E1102)